README.md

```diff
@@ -38,7 +39,7 @@ An `nvmrc` file is included if using [nvm](https://github.com/creationix/nvm).
 
 - Node 10.15.3
 
-## API
+## API (Using MongoDB)
```

Comment:
is this supposed to be mySQL?

Reply:
The content is currently for Mongo, but I should change it to reflect MySQL lol. I will do that now!